src/components/tabs/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState, forwardRef } from 'react'
+import PropTypes from 'prop-types'
+
+import Panel from './panel'
+import { makeStyles } from '../../utils/make-styles'
+
+
+const Tabs = forwardRef(({
+  classes = {
+    root: '',
+    header: '',
+    headerText: '',
+    selectedTab: '',
+    panel: '',
+  },
+  children,
+  defaultSelected,
+  ...props
+}, ref) => {
+  const [selected, setSelected] = useState(defaultSelected)
+
+  const styles = makeStyles({
+    tabsContainer: {
+      paddingBottom: '0.563rem',
+      borderBottom: '0.188rem solid #EAEAEA',
+    },
+    selectedPanel: {
+      textDecoration: 'underline',
+      textUnderlineOffset: '0.923rem',
+      textDecorationThickness: '0.188rem',
+      textDecorationColor: '#F6B747',
+    },
+  })
+
+  const tabsClasses = Object.freeze({
+    root: `tabs__root ${classes.root}`,
+    header: `tabs-header__container ${classes.header} ${styles.tabsContainer} flex flex-row gap-10`,
+    headerText: `tabs-header-text__container ${classes.headerText}`,
+    selectedTab: `tabs-selected__container ${classes.selectedTab} ${styles.selectedPanel}`,
+    panel: `tabs-panel__container ${classes.panel}`,
+  })
```

Comment:
lets keep the same naming format {component}__{container name} 

tabs__header-container
tabs__header-text-container

and so on